saleor/dashboard/order/utils.py

```diff
@@ -10,10 +10,10 @@
 
 
 def get_statics_absolute_url(request):
-    site = get_site_settings_from_request(request)
+    site_settings = get_site_settings_from_request(request)
```

Comment:
In this function you need to get the domain from Site model. You can fetch the Site instance directly, without accessing it through `site_settings`.

Reply:
Done